src/extension.ts

```diff
@@ -198,6 +201,8 @@ export async function activate(context: vscode.ExtensionContext) {
         const channelLogger = getChannelLogger(toolkitOutputChannel)
         channelLogger.error('AWS.channel.aws.toolkit.activation.error', 'Error Activating AWS Toolkit', error as Error)
         throw error
+    } finally {
+        recordToolkitInitialization(activationStartedOn)
```

Comment:
I'm thinking it might make sense to *only* record this metric if the toolkit activated successfully (or report activation status as well)

If we get a bunch of failures, it could skew this metric and hurt our ability to look at the low end.

Reply:
I agree, this metric should be based on the success case. Activation status should be emitted as a separate concern if we're interested in that. Will change